### PR TITLE
[misc] MAISTRA-2060 Point to Red Hat Jira for issues and disable other issue types

### PR DESCRIPTION
**Pull request by jewertow**
Cherry-pick 9377a6939b.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
+blank_issues_enabled: false
+
 contact_links:
-- name: "Crash bug"
-  url: https://istio.io/about/security-vulnerabilities/
-  about: "Please file any bug causing a crash to istio-security-vulnerability-reports@googlegroups.com."
+- name: "Issues"
+  url: https://issues.redhat.com/browse/MAISTRA
```
Comment (dgn) [inline .github/ISSUE_TEMPLATE/config.yml]: can you replace this with `OSSM`? We migrated away from `MAISTRA`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Istio
 
+## Issues for this repository are disabled
+
+Issues for this repository are tracked in Red Hat Jira. Please head to <https://issues.redhat.com/browse/MAISTRA> in order to browse or open an issue.
```
Comment (dgn) [inline README.md]: same here

Comment (jewertow) [reply README.md]: Fixed

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (jewertow) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed

**Review by dgn — APPROVED** [review]

**Pull request merged**